recipes-nodes/lighthouse/lighthouse.inc

```diff
@@ -1,16 +1,105 @@
 SUMMARY = "Lighthouse"
-DESCRIPTION = "geth is the command line interface for running a full ethereum node implemented in Go."
-HOMEPAGE = "https://geth.ethereum.org"
-LICENSE="CLOSED"
-LIC_FILES_CHKSUM=""
+DESCRIPTION = "Lighthouse is an Ethereum consensus client written in Rust"
+HOMEPAGE = "https://lighthouse.sigmaprime.io/"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=139bc4b5f578ecacea78dc7b7ad3ed3c"
 
+# Inherit the cargo_bin class to handle Rust/Cargo builds
 inherit cargo_bin
 
-# Enable network for the compile task allowing cargo to download dependencies
+# Define dependencies
+DEPENDS += "openssl-native zlib-native postgresql"
+
+# Set the source directory
+S = "${WORKDIR}/git"
+
+# Specify the toolchain to use
+TOOLCHAIN = "clang"
+
+# Rust flags for build configuration
+# Disable CPU-specific optimizations for better portability
+RUSTFLAGS += "-C target-cpu=generic"
+
+# Disable Link Time Optimization
+RUSTFLAGS += "-C lto=off"
```

Comment:
I wonder why?

Reply:
the `target-cpu=generic` is to make it more portable and reproducible.
It would support wider range of CPU types/architecture (maybe not relevant for our use-case since we only deploy for x86-64). 
However, it also helps with reproducibility as it would avoid CPU specific optimization that are supported on the build machine. Hence, different build machine should output same binary regardless of the build machine's CPU architecture.

Regarding `RUSTFLAGS += "-C lto=off"`, there is a trade-off here. Although these optimization could be beneficial but they might introduce non-determinism in the build process -> hindering the build reproducibility.
If you believe it is fine for this specific use-case, I can rollback those changes